README.md

```diff
@@ -1,2 +1,55 @@
 # reusable-p2p
```

Comment:
Let's also mention in the read me this is specifically for the CECG IDP, as it does the login etc

Reply:
Done!